ch_3_AppDev_Basic/3_Baichuan2_BasicApp.ipynb

```diff
@@ -0,0 +1,204 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Notebook 3: Basic Application Development On Baichuan2\n",
+    "\n",
+    "This notebook introduces the essential usage of `ipex-llm`, and walks you through building a very basic chat application built upon `Baichuan2`.\n",
+    "\n",
+    "## 3.1 Install `ipex-llm`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you haven't installed `ipex-llm`, install it as shown below. The one-line command will install the latest `ipex-llm` with all the dependencies for common LLM application development."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install --pre --upgrade ipex-llm[all]"
+   ]
```

Comment:
@plusbang  Should we also add a notebox for how to install on Linux OS?

Reply:
See here for more info: https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Overview/install_cpu.html#quick-installation

Reply:
> @plusbang Should we also add a notebox for how to install on Linux OS?

yeah, I think we could add this as a note.